shared/common-adapters/text.desktop.js

```diff
@@ -26,48 +49,69 @@ export default class Text extends Component {
       ...this.props.style
     }
 
-    const terminalPrefix = {
-      TerminalEmpty: <span>&nbsp;</span>,
-      TerminalCommand: <span>> </span>,
-      TerminalComment: <span># </span>
-    }[this.props.type]
+    const terminalPrefix = this._terminalPrefix(this.props.type)
+
+    if (this.props.inline) {
+      return <span className={this.props.link ? 'hover-underline' : ''} style={style} onClick={this.props.onClick}>{terminalPrefix}{this.props.children}</span>
```

Comment:
i really dislike 99% copy and paste code. It'd be better to just call createElement and pass in span/p i think